python/plugins/processing/tests/testdata/gdal_algorithm_raster_tests.yaml

```diff
@@ -214,8 +217,32 @@ tests:
         hash: 1c3adf57a7e319582736a1a222211e2360115fc930d6a12415dbdec3
         type: rasterhash
 
+  - algorithm: gdal:gridaverage
+    name: Moving average interpolation (gdal_grid) (GDAL >= 3.6)
+    condition:
+      gdal:
+        at_least: 3060000
+    params:
+      ANGLE: 0.0
+      DATA_TYPE: 5
+      INPUT:
+        name: pointsz.gml
+        type: vector
+      MIN_POINTS: 0
+      NODATA: 0.0
+      OPTIONS: ''
+      RADIUS_1: 5.0
+      RADIUS_2: 5.0
+    results:
+      OUTPUT:
+        hash: 362fe6834eab26e44e65c0af786cc2a6d82c99d468f365afb1434796
```

Comment:
'hash' accepts a list of hashes -- you could just add the new hash values to the existing tests 

(Honestly, these test results are rather meaningless now, so we don't need to worry about loss of contextual information like the version of gdal used to generate the hashes -- the actual real testing is done by checking the command line string  in GdalAlgorithmsRasterTest.py, and these tests only serve the purpose of checking that the command is actually run)

Reply:
Was about to write the same. I don't see how we can benefit from those changes.